scripts/start.js

```diff
@@ -365,10 +375,29 @@ function getUserInfo () {
         return reject(err)
       }
       if (body['access_token']) {
-        resolve({
-          userName: username.slice(0, username.indexOf('@')),
-          ssoToken: body.access_token,
-          domain: domain
+        request(`${engineUrl}api/users`, { json: true, strictSSL: false, headers: { Authorization: `Bearer ${body['access_token']}` } }, (userErr, userResponse, userBody) => {
```

Comment:
Could we have a comment here on why this request may not always contain data about current user?